### PR TITLE
Adds a click to copy button in code blocks

### PR DESCRIPTION
**Pull request by RohanGoyalDev**

### CODE REVIEW

File: source/conf.py [hunk 1]
```diff
@@ -132,6 +135,14 @@
...
 html_logo = '_static/logo.png'
 
+copybutton_image_svg = """
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-clipboard2-plus" viewBox="0 0 16 16">
+  <path d="M9.5 0a.5.5 0 0 1 .5.5.5.5 0 0 0 .5.5.5.5 0 0 1 .5.5V2a.5.5 0 0 1-.5.5h-5A.5.5 0 0 1 5 2v-.5a.5.5 0 0 1 .5-.5.5.5 0 0 0 .5-.5.5.5 0 0 1 .5-.5h3Z"/>
+  <path d="M3 2.5a.5.5 0 0 1 .5-.5H4a.5.5 0 0 0 0-1h-.5A1.5 1.5 0 0 0 2 2.5v12A1.5 1.5 0 0 0 3.5 16h9a1.5 1.5 0 0 0 1.5-1.5v-12A1.5 1.5 0 0 0 12.5 1H12a.5.5 0 0 0 0 1h.5a.5.5 0 0 1 .5.5v12a.5.5 0 0 1-.5.5h-9a.5.5 0 0 1-.5-.5v-12Z"/>
+  <path d="M8.5 6.5a.5.5 0 0 0-1 0V8H6a.5.5 0 0 0 0 1h1.5v1.5a.5.5 0 0 0 1 0V9H10a.5.5 0 0 0 0-1H8.5V6.5Z"/>
+</svg>
+"""
+
```
Comment (kezz) [inline source/conf.py:145]: This icon is MIT licensed from bootstrap - we'd need to include that license when using this, or find some other CC0 icon for example.

**Review by kezz — CHANGES_REQUESTED** [review]
Thanks for the PR! This is a great addition. I just have a small issue with the icon used, but otherwise this LGTM!

File: source/conf.py (hunk 1 above)
Comment (zml2008) [reply source/conf.py:145]: Can we also break this icon out into a separate file, rather than having it as a text blob here?

**Review by zml2008 — CHANGES_REQUESTED** [review]

**Comment by RohanGoyalDev** [comment]
Sorry I messed up the git rebase

**Pull request closed without merging**